### PR TITLE
GitHub Actions - Build Quarkus master

### PR DESCRIPTION
**Pull request by rsvoboda**
GitHub Actions - Build Quarkus master

### CODE REVIEW

**Comment by rsvoboda** [comment]
@Ladicek fyi

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -21,6 +21,8 @@ jobs:
         uses: joschi/setup-jdk@v1.0.0
         with:
           java-version: openjdk${{ matrix.java }}
+      - name: Build Quarkus master
+        run: git clone https://github.com/quarkusio/quarkus.git && cd quarkus && mvn -B clean install -DskipTests -DskipITs
```
Comment (Ladicek) [inline .github/workflows/ci.yml]: Do we need to `cd ..` for the next action, or does GitHub do that automatically?

Comment (rsvoboda) [reply .github/workflows/ci.yml]: It starts from the root

**Comment by Ladicek** [comment]
Works perfectly now, thanks!

**Pull request merged**